libs/wingsdk/src/target-awscdk/test-runner.ts

```diff
@@ -0,0 +1,92 @@
+import { CfnOutput, Lazy } from "aws-cdk-lib";
+import { Construct } from "constructs";
+import { Function as AwsFunction } from "./function";
+import * as cloud from "../cloud";
+import * as core from "../core";
+import { IInflightHost } from "../std";
+
+const OUTPUT_TEST_RUNNER_FUNCTION_ARNS = "WINGTESTRUNNERFUNCTIONARNS";
```

Comment:
Let's make this name more readable
```suggestion
const OUTPUT_TEST_RUNNER_FUNCTION_ARNS = "WingTestRunnerFunctionArns";
```